docs/usage/config-presets.md

```diff
@@ -232,6 +232,25 @@ Parameters are supported similar to other methods:
 }
 ```
 
+## Templating presets
+
+In order to provide flexible preset configurations, Renovate provides support using [templates](./templates.md) for preset names.
```

Comment:
We use the word provide twice in the same sentence. Also, after reading, it's not clear what the reader can _do_ with this feature, or why they should care.

I'd like something like this:

You can use x to be flexible with your presets.
This can be handy when you want to do y.
Read link to docs to learn more.

Note how I'm using the word `you` to put the reader into the text.

Reply:
Thanks for the valuable feedback 🙌  I've applied your suggestions, please have a look.